enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -67,7 +67,7 @@ confusing if a separate database on the same Postgresql database be used.
 | Column     | Description                                                         |
 | ---------- | ------------------------------------------------------------------- |
 | id         | an internal primary key                                             |
-| name       | a string column for the name of the parent replicated policy        |
+| name       | a string column for the name of the parent root policy              |
```

Comment:
Just the name of the root policy itself, not the `<namespace>.<name>` thing the propagator makes?

Just considering that on the managed cluster, I think I could have two different policies: say `ns-a.important-roles` and `ns-b.important-roles`, which could be different. It probably wouldn't be helpful to lose that namespace information when the event gets into this history?